cli.go

```diff
@@ -112,7 +112,7 @@ func main() {
 		},
 		cli.StringFlag{
 			Name: "dns",
-			Usage: "Enable the DNS challenge for solving using a provider." +
+			Usage: "Solve a DNS challenge using the specified provider." +
```

Comment:
I went for brevity: "Disables all other solvers". It's a really long line already. 3bceed427a63ab1789d7aa87e0d2360bc14da8c6